src/update.c

```diff
@@ -430,6 +436,7 @@ static int main_update()
 	 * First load up the old (current) manifests. Statedir could have been cleared
 	 * or corrupt, so don't assume things are already there. Updating subscribed
 	 * manifests is done as part of recurse_manifest */
+	link_submanifests(current_manifest, server_manifest, initial_subs, initial_subs, false);
```

Comment:
To find where a delta manifest is extracted to inside the state dir, I need to know its corresponding peer bundle's last change. The link_submanifests function must be called earlier so that the peer is set before I look for the delta manifest.

The peer->last_change is used to set the 'delta_manifest_state_dir' variable in load_manifests.

Reply:
I could move this above the comment though...